src/WormholeEndpoint.sol

```diff
@@ -8,19 +8,6 @@ import "./interfaces/IWormhole.sol";
 import "./Endpoint.sol";
 
 abstract contract WormholeEndpoint is Endpoint {
-    // TODO -- fix this after some testing
-    uint256 constant _GAS_LIMIT = 500000;
-
-    address immutable _wormholeCoreBridge;
-    address immutable _wormholeRelayerAddr;
-    uint256 immutable _wormholeEndpoint_evmChainId;
```

Comment:
Why have you modified these constants/immutables to now be in storage? I don't see why we'd need to do this + there are no setters to update these anyway